default.nix

```diff
@@ -125,12 +119,12 @@ let
   nix-tools = rec {
     # Programs for generating nix haskell package sets from cabal and
     # stack.yaml files.
-    package = nixToolsDeprecation (haskell { pkgs = commonLib.pkgsDefault; }).nix-tools;
+    package = nixToolsDeprecation pkgsDefault.haskell-nix.nix-tools;
     # A different haskell infrastructure
-    haskell = { pkgs }: nixToolsDeprecation import sources.haskell { inherit pkgs; };
+    haskell = _: nixToolsDeprecation (commonLib.getPkg {}).haskell-nix;
     # Script to invoke nix-tools stack-to-nix on a repo.
     regeneratePackages = nixToolsDeprecation commonLib.pkgsDefault.callPackage ./nix-tools-regenerate.nix {
```

Comment:
lets rename this regenerateStackPackages and the deprecation can be removed since the wallet team is utilizing the stack based workflow.